src/styles/index.css

```diff
@@ -124,15 +124,14 @@ pre > code {
 
 /* styles all tabs */
 [data-reach-tabs] [data-reach-tab-list] {
-  @apply bg-transparent font-semibold space-x-1;
+  @apply space-x-1 font-semibold bg-transparent;
 }
-
 [data-reach-tabs] [data-reach-tab] {
-  @apply px-4 py-3 text-base text-gray-700 dark:text-gray-300 rounded-t-lg hover:bg-gray-50 dark:hover:bg-black transition-colors ease-in-out duration-300;
```

Comment:
will these global changes need to be verified?

Reply:
it's just reordered. I should have disabled that plug-in, my bad.

Reply:
Headwind — https://github.com/heybourn/headwind